ios/Podfile.lock

```diff
@@ -2766,6 +2766,6 @@ SPEC CHECKSUMS:
   Yoga: 86fed2e4d425ee4c6eab3813ba1791101ee153c6
   ZXingObjC: 8898711ab495761b2dbbdec76d90164a6d7e14c5
 
-PODFILE CHECKSUM: 2e2091adf6eb0d6921acab51e937558fd3a0e63d
+PODFILE CHECKSUM: 548074f79b4144edeae371a061bbe3a1aa0d8909
```

Comment:
do you have any insight for the podfile checksum change where you didn't change the podfile?

Reply:
I'm not 100% sure to be honest, at first I thought this could be related to my switching between so many branches but I tried reverting this and every time I run `pod install` even after deleting the `Pods` folder the checksum output is always `548074f79b4144edeae371a061bbe3a1aa0d8909 `

Reply:
well np! just a curious question. feel free to merge it and we will know after merged.